crates/bevy_render/Cargo.toml

```diff
@@ -79,3 +79,16 @@ encase = { version = "0.5", features = ["glam"] }
 # For wgpu profiling using tracing. Use `RUST_LOG=info` to also capture the wgpu spans.
 profiling = { version = "1", features = ["profile-with-tracing"], optional = true }
 async-channel = "1.8"
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+js-sys = "0.3"
+web-sys = { version = "0.3", features = [
+    'Blob',
+    'Document',
+    'Element',
+    'HtmlElement',
+    'Node',
+    'Url',
+    'Window',
```

Comment:
Will including more web-sys features meaningfully affect our wasm build sizes?

Reply:
no, they are already enabled by other dependencies...

The current list of `web-sys` features enabled: AddEventListenerOptions, AngleInstancedArrays, AudioBuffer, AudioBufferSourceNode, AudioContext, AudioContextOptions, AudioContextState, AudioDestinationNode, AudioNode, AudioScheduledSourceNode, BaseAudioContext, BeforeUnloadEvent, CssStyleDeclaration, Document, DomException, DomRect, DomRectReadOnly, Element, Event, EventListenerOptions, EventTarget, ExtBlendMinmax, ExtColorBufferFloat, ExtColorBufferHalfFloat, ExtDisjointTimerQuery, ExtFragDepth, ExtSRgb, ExtShaderTextureLod, ExtTextureFilterAnisotropic, FocusEvent, Gamepad, GamepadButton, GamepadMappingType, Gpu, GpuAdapter, GpuAddressMode, GpuAutoLayoutMode, GpuBindGroup, GpuBindGroupDescriptor, GpuBindGroupEntry, GpuBindGroupLayout, GpuBindGroupLayoutDescriptor, GpuBindGroupLayoutEntry, GpuBlendComponent, GpuBlendFactor, GpuBlendOperation, GpuBlendState, GpuBuffer, GpuBufferBinding, GpuBufferBindingLayout, GpuBufferBindingType, GpuBufferDescriptor, GpuCanvasAlphaMode, GpuCanvasConfiguration, GpuCanvasContext, GpuColorDict, GpuColorTargetState, GpuCommandBuffer, GpuCommandBufferDescriptor, GpuCommandEncoder, GpuCommandEncoderDescriptor, GpuCompareFunction, GpuCompilationInfo, GpuCompilationMessage, GpuCompilationMessageType, GpuComputePassDescriptor, GpuComputePassEncoder, GpuComputePipeline, GpuComputePipelineDescriptor, GpuCullMode, GpuDepthStencilState, GpuDevice, GpuDeviceDescriptor, GpuDeviceLostInfo, GpuDeviceLostReason, GpuError, GpuErrorFilter, GpuExtent3dDict, GpuFeatureName, GpuFilterMode, GpuFragmentState, GpuFrontFace, GpuImageCopyBuffer, GpuImageCopyExternalImage, GpuImageCopyTexture, GpuImageCopyTextureTagged, GpuImageDataLayout, GpuIndexFormat, GpuLoadOp, GpuMipmapFilterMode, GpuMultisampleState, GpuObjectDescriptorBase, GpuOrigin2dDict, GpuOrigin3dDict, GpuOutOfMemoryError, GpuPipelineDescriptorBase, GpuPipelineLayout, GpuPipelineLayoutDescriptor, GpuPowerPreference, GpuPrimitiveState, GpuPrimitiveTopology, GpuProgrammableStage, GpuQuerySet, GpuQuerySetDescriptor, GpuQueryType, GpuQueue, GpuRenderBundle, GpuRenderBundleDescriptor, GpuRenderBundleEncoder, GpuRenderBundleEncoderDescriptor, GpuRenderPassColorAttachment, GpuRenderPassDepthStencilAttachment, GpuRenderPassDescriptor, GpuRenderPassEncoder, GpuRenderPipeline, GpuRenderPipelineDescriptor, GpuRequestAdapterOptions, GpuSampler, GpuSamplerBindingLayout, GpuSamplerBindingType, GpuSamplerDescriptor, GpuShaderModule, GpuShaderModuleDescriptor, GpuStencilFaceState, GpuStencilOperation, GpuStorageTextureAccess, GpuStorageTextureBindingLayout, GpuStoreOp, GpuSupportedFeatures, GpuSupportedLimits, GpuTexture, GpuTextureAspect, GpuTextureBindingLayout, GpuTextureDescriptor, GpuTextureDimension, GpuTextureFormat, GpuTextureSampleType, GpuTextureView, GpuTextureViewDescriptor, GpuTextureViewDimension, GpuUncapturedErrorEvent, GpuUncapturedErrorEventInit, GpuValidationError, GpuVertexAttribute, GpuVertexBufferLayout, GpuVertexFormat, GpuVertexState, GpuVertexStepMode, HtmlCanvasElement, HtmlElement, HtmlImageElement, HtmlMediaElement, HtmlVideoElement, ImageBitmap, ImageBitmapRenderingContext, KeyboardEvent, MediaQueryList, MediaQueryListEvent, MouseEvent, Navigator, Node, NodeList, OesElementIndexUint, OesStandardDerivatives, OesTextureFloat, OesTextureFloatLinear, OesTextureHalfFloat, OesTextureHalfFloatLinear, OesVertexArrayObject, OffscreenCanvas, OvrMultiview2, Performance, PerformanceTiming, PointerEvent, Request, Response, UiEvent, VideoFrame, WebGl2RenderingContext, WebGlActiveInfo, WebGlBuffer, WebGlFramebuffer, WebGlProgram, WebGlQuery, WebGlRenderbuffer, WebGlRenderingContext, WebGlSampler, WebGlShader, WebGlSync, WebGlTexture, WebGlTransformFeedback, WebGlUniformLocation, WebGlVertexArrayObject, WebglColorBufferFloat, WebglCompressedTextureAstc, WebglCompressedTextureEtc, WebglCompressedTextureEtc1, WebglCompressedTexturePvrtc, WebglCompressedTextureS3tc, WebglCompressedTextureS3tcSrgb, WebglDebugRendererInfo, WebglDebugShaders, WebglDepthTexture, WebglDrawBuffers, WebglLoseContext, WheelEvent, Window, WorkerGlobalScope, WorkerNavigator, console, gpu_map_mode

Reply:
Just tried a build, it's 14kB smaller with this PR. Build duration were similar

Reply:
Amazing that it got smaller by adding features. Works for me